preview-src/docs-ndl.adoc

```diff
@@ -36,7 +36,7 @@ image:icons/ndl/deployment-options.svg[]
 [.description]
 Choose from self-managed local and cloud deployments, or fully-managed options.
 
-[.link]
+[.link.display]
```

Comment:
@MarkWoulfeNeo4j if we merge this change, then you'll need to update GTM with this new subclass of links in cards in the homepage, I think